pkg/controller/pxc/upgrade.go

```diff
@@ -172,7 +172,8 @@ func (r *ReconcilePerconaXtraDBCluster) smartUpdate(sfs api.StatefulApp, cr *api
 	}
 
 	if sfs.StatefulSet().Status.ReadyReplicas < sfs.StatefulSet().Status.Replicas {
-		return fmt.Errorf("can't start/continue 'SmartUpdate': waiting for all replicas are ready")
+		log.Info("can't start/continue 'SmartUpdate': waiting for all replicas are ready")
```

Comment:
should it be also `log.Info` if the backup is running in the line 170

Reply:
It can return real error witch should be logged as error, I think

Reply:
it depends on error, just running backup is not the real error, it is just waiting till the moment when the backup is finished

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/573/commits/3126ec3c6ea3e359a3c42b55104b7414f9b4e509